api-reference/transaction-intent.md

```diff
@@ -8,25 +8,24 @@ An TransactionIntent to create an Ethereum transaction on, or through, an Aragon
 
 Get all the possible transaction paths for a given address. This can be useful to let users pick the one they want. Otherwise, `TransactionIntent#transactions()` can get called directly.
 
-| Name | Type | Description |
-| :--- | :--- | :--- |
-| `account` | `String` | The account that will sign the transaction. |
-| `options` | `Object` | Options object. |
-| `options.as` | `String` | Address of an Aragon organization, or its agent app, through which the paths should get created. |
-| `options.path` | `String[]` | An array of address that conform a transaction path, it will be verified without calculating other paths. |
-| returns | `TransactionPath[]` | Array of all the possible transaction paths. |
+| Name           | Type                         | Description                                                                                               |
+| :------------- | :--------------------------- | :-------------------------------------------------------------------------------------------------------- |
+| `account`      | `String`                     | The account that will sign the transaction.                                                               |
+| `options`      | `Object`                     | Options object.                                                                                           |
+| `options.as`   | `String`                     | Address of an Aragon organization, or its agent app, through which the paths should get created.          |
+| `options.path` | `String[]`                   | An array of address that conform a transaction path, it will be verified without calculating other paths. |
+| returns        | `Promise<TransactionPath[]>` | Array of all the possible Transaction Paths.                                                              |
```

Comment:
For clarity, I think we should either use a descriptive name (“transaction paths”) or the exact class name (`TransactionPath`) in the documentation.

```suggestion
| returns        | `Promise<TransactionPath[]>` | Array of all the possible transaction paths.                                                              |
```